confs/.pre-commit-config.yaml

```diff
@@ -1,16 +1,4 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 21.12b0
-    hooks:
-      - id: black
-        name: Black
-        language_version: python3
-  - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.4
-    hooks:
-      - id: flake8
-        name: Flake
-        language_version: python3
   - repo: https://github.com/jazzband/pip-tools
     rev: 6.1.0
     hooks:
```

Comment:
as outcome from issue #19 this file will be removed in the pending PR #24

Reply:
Sure, atm I've just removed what had to go in ci pipeline